x/pocketcore/keeper/session.go

```diff
@@ -10,16 +10,21 @@ import (
 // "HandleDispatch" - Handles a client request for their session information
 func (k Keeper) HandleDispatch(ctx sdk.Ctx, header types.SessionHeader) (*types.DispatchResponse, sdk.Error) {
 	// retrieve the latest session block height
-	latestSessionBlockHeight := k.GetLatestSessionBlockHeight(ctx)
-	// set the session block height
-	header.SessionBlockHeight = latestSessionBlockHeight
+	var sessionBlockHeight int64
```

Comment:
I don't think we shold be touching this part of the code at all. Handling the relay with validate the entire proof.

Leaving dispatching as is.

Reply:
I modified dispatch with additional functionality, so that way portals can dispatch and retrieve information about old sessions. This is important if the portal wants to account for node's are still behind. It's optional, and coded as such.

Otherwise, there is no RPC call to get app's session information about a certain block height except the latest, which is counterproductive to adding a tolerance.